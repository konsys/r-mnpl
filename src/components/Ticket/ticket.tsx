import React from "react";
export const Ticket = (props: any) => (
  <>
    <div className="table-body-board-ticket" style={{ display: "none" }}>
      <div className="table-body-board-ticket-side">
        <div>
          <img src="//m1.dogecdn.wtf/tickets/m1cup-2018.svg" />
        </div>
        <div className="_button">
          <input
            type="button"
            className="button button-free button-small"
            value="Купить за 59 р."
          />
        </div>
      </div>
      <div className="table-body-board-ticket-text">
        <div className="_title">Билет на M1 Cup 2018</div>
        <div className="_text">
          Купив и активировав этот билет, вы получите шанс на получение
          сувенирных кейсов во время просмотра матчей M1 Cup.
        </div>
      </div>
    </div>
  </>
);
