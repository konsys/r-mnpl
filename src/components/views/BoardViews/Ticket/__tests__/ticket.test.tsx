import React from "react";
import { Ticket } from "../Ticket";
import { shallow } from "enzyme";

describe("Table helper  test", () => {
  beforeEach(() => jest.clearAllMocks());
  it("should render", () => {
    expect(shallow(<Ticket />)).toMatchSnapshot();
  });
});
