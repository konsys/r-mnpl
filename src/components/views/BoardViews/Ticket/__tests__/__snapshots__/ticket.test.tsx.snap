// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table helper  test should render 1`] = `ShallowWrapper {}`;
